diff -up ffmpeg-5.0/libavformat/avformat.h.accessor~ ffmpeg-5.0/libavformat/avformat.h
--- ffmpeg-5.0/libavformat/avformat.h.accessor~	2022-01-15 18:58:45.604804187 +0100
+++ ffmpeg-5.0/libavformat/avformat.h	2022-01-15 19:00:10.496181433 +0100
@@ -2890,4 +2890,9 @@ AVRational av_stream_get_codec_timebase(
  * @}
  */
 
+
+int64_t av_stream_get_cur_dts(AVStream *s);
+int64_t av_stream_get_first_dts(AVStream *s);
+int av_stream_get_pts_wrap_bits(AVStream *s);
+int64_t av_stream_get_codec_info_nb_frames(AVStream *s);
 #endif /* AVFORMAT_AVFORMAT_H */
diff -up ffmpeg-5.0/libavformat/utils.c.accessor~ ffmpeg-5.0/libavformat/utils.c
--- ffmpeg-5.0/libavformat/utils.c.accessor~	2022-01-15 18:58:45.604804187 +0100
+++ ffmpeg-5.0/libavformat/utils.c	2022-01-15 19:00:52.529369158 +0100
@@ -102,6 +102,26 @@ void avpriv_stream_set_need_parsing(AVSt
     ffstream(st)->need_parsing = type;
 }
 
+int64_t av_stream_get_cur_dts(AVStream *s)
+{
+    return ffstream(s)->cur_dts;
+}
+
+int64_t av_stream_get_first_dts(AVStream *s)
+{
+    return ffstream(s)->first_dts;
+}
+
+int av_stream_get_pts_wrap_bits(AVStream *s)
+{
+    return s->pts_wrap_bits;
+}
+
+int64_t av_stream_get_codec_info_nb_frames(AVStream *s)
+{
+    return ffstream(s)->codec_info_nb_frames;
+}
+
 void av_format_inject_global_side_data(AVFormatContext *s)
 {
     FFFormatContext *const si = ffformatcontext(s);
