diff -uprN ffmpeg-9.0.1/libavformat/avformat.h ffmpeg-9.0.1/libavformat/avformat.h
--- ffmpeg-9.0.1/libavformat/avformat.h	2026-08-04 00:14:48.000000000 +0200
+++ ffmpeg-9.0.1/libavformat/avformat.h	2026-08-12 17:41:56.876106893 +0200
@@ -3189,4 +3189,9 @@ int avformat_queue_attached_pictures(AVF
  * @}
  */
 
+
+int64_t av_stream_get_cur_dts(AVStream *s);
+int64_t av_stream_get_first_dts(AVStream *s);
+int av_stream_get_pts_wrap_bits(AVStream *s);
+int64_t av_stream_get_codec_info_nb_frames(AVStream *s);
 #endif /* AVFORMAT_AVFORMAT_H */
diff -uprN ffmpeg-9.0.1/libavformat/utils.c ffmpeg-9.0.1/libavformat/utils.c
--- ffmpeg-9.0.1/libavformat/utils.c	2026-08-04 00:14:48.000000000 +0200
+++ ffmpeg-9.0.1/libavformat/utils.c	2026-08-12 17:41:56.876689641 +0200
@@ -710,3 +710,23 @@ void *ff_bprint_finalize_as_fam(struct A
 
     return p;
 }
+
+int64_t av_stream_get_cur_dts(AVStream *s)
+{
+    return ffstream(s)->cur_dts;
+}
+
+int64_t av_stream_get_first_dts(AVStream *s)
+{
+    return ffstream(s)->first_dts;
+}
+
+int av_stream_get_pts_wrap_bits(AVStream *s)
+{
+    return s->pts_wrap_bits;
+}
+
+int64_t av_stream_get_codec_info_nb_frames(AVStream *s)
+{
+    return ffstream(s)->codec_info_nb_frames;
+}
